extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -81,6 +104,9 @@ oppia.directive('oppiaInteractiveGraphInput', [
 
           var updateGraphFromJSON = function(jsonGraph) {
             var newGraph = HtmlEscaperService.escapedJsonToObj(jsonGraph);
+            if (!$scope.interactionIsActive) {
```

Comment:
I think this is in the wrong place. In the "interactionIsActive" case this shouldn't be called at all, right?

Reply:
There was an option to give an initial graph in create exploration, in which case, this would be called even if interaction is active.

Reply:
Then shouldn't the "if" check be in the caller of this function?

Reply:
$scope.getLastAnswer() can be directly used to create new graph but in the other one, HtmlEscaperService.escapedJsonToObj() is used on top of raw data, thats why I put if condition inside function.